eng/pipelines/templates/steps/sync-repo-merge-branch.yml

```diff
@@ -74,11 +77,14 @@ steps:
         Write-Host "git push origin $mergeBranch -f"
         git push origin $mergeBranch -f
 
-        Write-Host "gh pr create --base ${{ target.key }} --head  --reviewer weshaggard --repo ${{ repo.key }} --body $commitMessage --title $commitMessage"
-        gh pr create --base ${{ target.key }} --head $mergeBranch --reviewer weshaggard --repo ${{ repo.key }} --body $commitMessage --title $commitMessage
+        Write-Host "gh pr create --base ${{ target.key }} --head  --repo ${{ repo.key }} --body $commitMessage --title $commitMessage"
```

Comment:
Do you no longer want to add yourself as reviewer for any sync PRs?

Reply:
I added that originally so I can keep an eye on them but the codeowners should do fine for our sync PRs.